load_site.py

```diff
@@ -0,0 +1,27 @@
+import asyncio
+import aiohttp
+import time
+
+
+async def load_yandex():
+    with aiohttp.ClientSession() as s:
+        resp = await s.get('https://ya.ru')
+        print(await resp.json())
+
+
+async def load_google():
+    with aiohttp.ClientSession() as s:
+        resp = await s.get('https://google.com')
+        print(await resp.json())
+
+
+async def main():
+    while True:
+        await load_yandex()
+        await load_google()
+
+        time.sleep(1)
```

Comment:
Блокирующий вызов `time.sleep` лучше сразу заменить на неблокирующий вызов [asyncio.sleep](https://docs.python.org/3/library/asyncio-task.html#asyncio.sleep)

```suggestion
        await asyncio.sleep(1)
```